.travis.yml

```diff
@@ -21,35 +16,44 @@ branches:
 matrix:
     fast_finish: true
     include:
-        # Test stable versions
+          # Test with lowest dependencies
+        - php: 7.1
```

Comment:
imho this should also be the oldest php version we support to really get the oldest possible thing